src/main/java/org/troyargonauts/subsystems/Limelight.java

```diff
@@ -0,0 +1,167 @@
+package org.troyargonauts.subsystems;
+
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import org.troyargonauts.Constants;
+
+/**
+ * Limelight code
```

Comment:
What does the limelight code do, assume the person reading doesn't know what Limelight is. Explain what kids of vision processing we are doing, and possibly a common use case. 2 sentence average.